service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/OtherProceedingsControllerTest.java

```diff
@@ -75,31 +75,32 @@ void shouldReturnNoErrorsWhenPOnGoingProceedingIsDontKnow() throws Exception {
 
     private Proceeding createProceeding(String onGoingProceeding) {
         return new Proceeding(onGoingProceeding, "", "", "", "",
-            "", "", "", "", "", "");
+                              "", "", "", "", "", "");
     }
 
     private AboutToStartOrSubmitCallbackResponse makeRequest(Proceeding proceeding) throws Exception {
-        HashMap<String, Object> map = MAPPER.readValue(MAPPER.writeValueAsString(proceeding),
-            new TypeReference<Map<String, Object>>() {
-            });
+        Map<String, Object> map = MAPPER.readValue(MAPPER.writeValueAsString(proceeding), new TypeReference<>() {
+        });
 
         CallbackRequest request = CallbackRequest.builder()
-            .caseDetails(CaseDetails.builder()
-                .id(12345L)
-                .data(ImmutableMap.<String, Object>builder().put("proceeding", map).build())
-                .build())
-            .build();
+                                                 .caseDetails(CaseDetails.builder()
+                                                                         .id(12345L)
+                                                                         .data(
+                                                                             ImmutableMap.<String, Object>builder().put(
+                                                                                 "proceeding", map).build())
+                                                                         .build())
+                                                 .build();
```

Comment:
What happened with the indents here? 😄

Reply:
Think that was when the checkstyle didn't import correctly on my machine, will fix

Reply:
fixed